pkg/osbuild/lvm2_lv_device.go

```diff
@@ -5,6 +5,8 @@ package osbuild
 type LVM2LVDeviceOptions struct {
 	// Logical volume to activate
 	Volume string `json:"volume"`
+	// The partitin the volume group is located on
+	VGPartnum *int `json:"vg_partnum,omitempty"`
```

Comment:
Can we get away with a concrete `int` (or `uint`) here since partition numbers are never 0 or would we rather not rely on 0 being "empty"?

Reply:
Yeah, I think that would work. I'm not sure about it though, I dislike the pointer but I do like that it's explicit about being optional (and here uniform/consistent with the rest of the way we flag things as optional) - I wonder if https://github.com/osbuild/images/pull/860 would help here and make it nicer?

Reply:
Let's keep the pointer then and push forward with the `types.Option` PR.